src/presentation/web/src/App.tsx

```diff
@@ -24,7 +24,7 @@ const App: React.FC = () => {
     return (
         <ThemeProvider theme={theme}>
             <MuiPickersUtilsProvider utils={DateFnsUtils} locale={plLocale}>
-                <Container style={{ marginTop: 64 }}>
```

Comment:
Wartość 64px to wysokość menu. Niestety nie znam innego eleganckiego rozwiązania, ale dzięki temu cały conntent strony jest równo pod navbarem